rotkehlchen/chain/evm/decoding/extrafi/balances.py

```diff
@@ -151,7 +151,7 @@ def _query_farm_positions(
                 if debt_amount == 0:
                     continue
 
-                amount = token_normalized_value(debt_amount, debt_token)
+                amount = asset_normalized_value(debt_amount, debt_token)
```

Comment:
since you touched so many places how about introducing kwargs in places they are missing it?